packages/specs/pages/index.mdx

```diff
@@ -1,16 +1,18 @@
-## Namada
+# Namada
 
-Welcome to the Namada specification!
+Welcome to the Namada specification 🟨⚫
 
 ## What is Namada?
 
-Namada is a sovereign proof-of-stake blockchain, using [CometBFT](https://cometbft.com/) (nee Tendermint) BFT consensus,
-which enables multi-asset private transfers for any native or non-native asset
-using a [multi-asset shielded pool](https://research.metastate.dev/multi-asset_shielded_pool/) derived from the [Sapling circuit](https://z.cash/upgrade/sapling/).
-Namada features full IBC protocol support, a natively integrated Ethereum bridge, a modern proof-of-stake system with automatic reward compounding and cubic slashing,
-a stake-weighted governance signalling mechanism, and a dual proactive/retroactive public goods funding system.
-Users of shielded transfers are rewarded for their contributions to the privacy set in the form of native protocol tokens.
-A multi-asset shielded transfer wallet is provided in order to facilitate safe and private user interaction with the protocol. Namada supports a feature called Shielded Actions, which allows users to hold their assets privately on Namada most of the time while occaisionally unshielding specific assets in order to interact with existing applications on transparent chains.
+Namada is a sovereign proof-of-stake blockchain, using the [CometBFT](https://cometbft.com/) (nee Tendermint) implementation of a Byzantine Fault Tolerance (BFT) consensus algorithm. Namada enables multi-asset private transfers for any native or non-native asset using a [multi-asset shielded pool](https://research.metastate.dev/multi-asset_shielded_pool/) derived from the [Sapling circuit](https://z.cash/upgrade/sapling/).
```

Comment:
```suggestion
Namada is a sovereign proof-of-stake blockchain, using the [CometBFT](https://cometbft.com/) (nee Tendermint) BFT consensus algorithm. Namada enables multi-asset private transfers for any native or non-native asset using a [multi-asset shielded pool](https://research.metastate.dev/multi-asset_shielded_pool/) derived from the [Sapling circuit](https://z.cash/upgrade/sapling/).
```

Reply:
otherwise this sentence is just too long. if people want to know what BFT means they can look it up :)